docs/source/advanced_usage.rst

```diff
@@ -425,6 +434,10 @@ We can run this program a few times to see what we get in the readout register `
      [1]
      [0]]
 
+Applying a numshots loop
+------------------------
+
+
```

Comment:
Supposed to be empty?

Reply:
Oops, artifact of me moving where the docs lived. Fixed.